### PR TITLE
Support updates

### PR DESCRIPTION
**Pull request by laboon**
Update wut@kusama.network to support@kusama.network

Use bulleted lists to separate entities - they were all on one line

### CODE REVIEW

**Comment by laboon** [comment]
Ah, forgot to run `prettier`

File: docs/kusama-index.md [hunk 1]
```diff
@@ -50,29 +50,28 @@ everything else.
...
-[Kusama newsletter](https://kusama.network/newsletter) [r/kusama](https://reddit.com/r/kusama)
-
-[kappa]: https://polkascan.io/pre/kusama/council/motion/94
-[thousand validators]: https://polkadot.network/join-kusamas-thousand-validators-programme/
-[kusama claims]: https://claim.kusama.network
+- [@kusamanetwork](https://twitter.com/kusamanetwork)
+- [Kusama newsletter](https://kusama.network/newsletter) [r/kusama](https://reddit.com/r/kusama)
+- [kappa]: https://polkascan.io/pre/kusama/council/motion/94
+- [thousand validators]: https://polkadot.network/join-kusamas-thousand-validators-programme/
+- [kusama claims]: https://claim.kusama.network
```
Comment (joepetrowski) [inline docs/kusama-index.md]: These are just link aliases and shouldn't have bullet points.

**Review by joepetrowski — APPROVED** [review]
Looks good just needs the fix at the bottom

File: docs/kusama-index.md [hunk 2]
```diff
@@ -50,29 +50,28 @@ everything else.
...
 ### Get updates
 
-[@kusamanetwork](https://twitter.com/kusamanetwork)
-[Kusama newsletter](https://kusama.network/newsletter) [r/kusama](https://reddit.com/r/kusama)
-
-[kappa]: https://polkascan.io/pre/kusama/council/motion/94
-[thousand validators]: https://polkadot.network/join-kusamas-thousand-validators-programme/
-[kusama claims]: https://claim.kusama.network
+- [@kusamanetwork](https://twitter.com/kusamanetwork)
+- [Kusama newsletter](https://kusama.network/newsletter) [r/kusama](https://reddit.com/r/kusama)
```
Comment (krichard410) [inline docs/kusama-index.md]: Should Kusama newsletter and the Kusama reddit page be on different bullet points?

**Review by krichard410 — APPROVED** [review]

**Review by lsaether — APPROVED** [review]

**Pull request merged**